src/components/Dashboard/TodoList.vue

```diff
@@ -1498,6 +1498,29 @@ export default class TodoList extends Vue {
     }
   }
 
+  // navigate to Create UI if Full/Limited restoration or to Edit UI if Limited extension/Full to Limited conversion
+  buildRestorationUrl (item: TodoItemIF): string {
+    let url: string
+    let restorationType: string
+
+    if (item.filingSubType === RestorationTypes.FULL || item.filingSubType === RestorationTypes.LIMITED) {
+      url = `${this.getCreateUrl}?id=${this.getIdentifier}`
+    }
+
+    if (item.filingSubType === RestorationTypes.LTD_EXTEND || item.filingSubType === RestorationTypes.LTD_TO_FULL) {
+      if (item.filingSubType === RestorationTypes.LTD_EXTEND) {
+        restorationType = RestorationTypes.LTD_EXTEND
+      }
+
+      if (item.filingSubType === RestorationTypes.FULL) {
+        restorationType = RestorationTypes.FULL
+      }
+
+      url = `${this.getEditUrl}${this.getIdentifier}/` + restorationType + `?restoration-id=${item.filingId}`
+    }
+    return url
+  }
+
   // this is called for both Resume Payment and Retry Payment
```

Comment:
I tried moving this to some common file as similar logic is used in 2 classes (ToDoList and StaffNotation).  After some thoughts, I decided to not to go with it because code was becoming unnecessarily long (for a simple change to add url) with so many parameters to pass to the function call. If we feel strongly, I can go ahead with that implementation. 
@jonathan-longe, I am holding off on cleaning up 'ToDoList.vue' file (reduce if-else and use dynamic component) after chatting with @severinbeauvais. He mentioned he is trying something similar in 'FilingHistory'. After that is done we can take this on.

Reply:
@ketaki-deodhar understood, I appreciate the note.  I'll doublecheck that we have a ticket for this refactoring.